src/customers/account-dashboard-order-by-sku.md

```diff
@@ -3,6 +3,13 @@ ee_only: true
 title: Order by SKU
 ---
 
+SKU stands for Stock Keeping Unit. SKUs help online sellers identify the most important product characteristics such as: size, color, price, material etc. Don't confuse product IDs with SKUs:
+
+    - The `Product ID` is a set of numbers that are used internally to identify products and are not available to customers.
+    - The `SKU` is generated by the seller based on the product name and attributes for marketing or internal tracking.
+
+Basically, an article number includes a set of abbreviations indicating the distinguishing characteristics of the product. The maximum SKU length in Magento 2 is 64 characters. SKUs are very important to effectively track and manage inventory, so setting them up correctly is critical for e-commerce.
```

Comment:
You haven't mentioned article number anywhere else and it may confuse things.
```suggestion
Normally, an SKU includes a set of abbreviations indicating the distinguishing characteristics of the product. The maximum SKU length in Magento 2 is 64 characters. SKUs are very important to effectively track and manage inventory, so setting them up correctly is critical for e-commerce.
```

Reply:
Thanks for your help🙏 
Done, please check)